### ISSUE TITLE
Propostas de Talks, Light Talks e Workshop para apresentar em Meetups do GDG

### ISSUE THREAD

**Issue by Lucasktrindade** [gdg-duque-de-caxias/gdg-duque-de-caxias.github.io#13]
Estou criando esta issue, para todos comentarem sobre o que podem falar e o que desejam e assim a gente discutir e poder criar encontros abordando temas de maneira mais especifica.

**Comment by Lucasktrindade** [comment]
Seria legal a gente ter alguém falando de Firebase. Eu consigo abranger tecnologias de front, posso falar sobre Metodologias CSS, Web Components com Polymer. E falar um pouco de Laravel, criar API com Silex, mostrar sobre Python e Django. E atualmente falar o básico de Ionic 2 e NG2.

**Comment by IgorHalfeld** [comment]
Eu poderia falar sobre qualquer coisa, acho legal que mesmo que não tenha conhecimento do tema fazer uma talk o/, mas já de começo eu queria propor uma sobre Graphql, RethinkDB ou "Design Patterns real life"

**Comment by Lucasktrindade** [comment]
Sim, você preparando o que vai falar aprende bastante sobre o assunto. E não precisa expert para falar. Esse de Design Patterns Real Life parece ser interessante.

**Comment by gabrielclima** [comment]
Eu tô afim de dar uma introdução de Golang. Falar um pouquinho das características, do porquê usar, em quais cases usar.

Fazer até um getting started básico com a galera.

**Comment by IgorHalfeld** [comment]
Achei daóra @gabrielclima é uma linguagem que tenho iniciado os estudos, uma talk amostrando esse _getting started_ seria loko

**Comment by gabrielclima** [comment]
A ideia é quebrar a barreira do universo! kkk

**Comment by puggianjv** [comment]
Seria legal a gente ter um pouco de conteúdo voltado para devOps. É dificuldade de muita gente e é um assunto que "tá na moda". Eu tô com docker na minha fila de estudo. Não agora no próximo, mas posso falar sobre ele futuramente.

**Comment by gabrielclima** [comment]
Show de bola @puggianjv 

Nós realmente não temos conteúdo de DevOps. Acho muito muito válido

**Comment by albertosouza** [comment]
Acho que posso apresentar um pouco de IoT e microlocalização com beacons
